NMS-15296: Update Fix Karaf script instruction in Housekeeping Tasks (#5818)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/admin/housekeeping.adoc

== After upgrading

=== Troubleshooting

If your {page-component-title} instance fails to start after upgrading, you can run the Fix Karaf script to delete cached items.
The script tries to fix configuration problems by pruning `/opt/opennms/data` and restoring all Karaf-related configuration files to a pristine state.

[source, console]
/opt/opennms/bin/fix-karaf-setup.sh

WARNING: The Fix Karaf script can be destructive to files that may contain user-specific configuration settings.

> It is recommended that you back up `/opt/opennms/etc` before running the script.

Reviewer: ```suggestion
We recommend that you back up `/opt/opennms/etc` before running the script.
```
